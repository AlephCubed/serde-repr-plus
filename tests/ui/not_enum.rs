use serde_repr_plus::Serialize_repr_clamp;

#[derive(Serialize_repr_clamp)]
struct SmallPrime {
    two: u8,
    three: u8,
    five: u8,
    seven: u8,
}

fn main() {}
