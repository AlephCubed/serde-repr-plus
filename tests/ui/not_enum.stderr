error: input must be an enum
 --> tests/ui/not_enum.rs:3:10
  |
3 | #[derive(Serialize_repr_clamp)]
  |          ^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Serialize_repr_clamp` (in Nightly builds, run with -Z macro-backtrace for more info)
